test/cmd/builds.sh

```diff
@@ -147,6 +147,33 @@ os::cmd::expect_success "oc delete bc/test-buildconfig"
 echo "set build-hook: ok"
 os::test::junit::declare_suite_end
 
+os::test::junit::declare_suite_start "cmd/builds/setbuildhook"
+# Validate the set build-secret command
+arg="-f test/testdata/test-bc.yaml"
```

Comment:
I'm not sure we gain a huge amount from this as we want our tests to be as literal as possible so they're easier to read